sdk/communication/communication-tests-template.yml

```diff
@@ -21,6 +21,9 @@ stages:
           groupId: com.azure
           safeName: ${{ parameters.SafeName }}
       ServiceDirectory: communication
+      PreSteps:
```

Comment:
What are these pre steps for?

Reply:
"Pre steps" is defined in live.tests.yml. In this change, we use it to add "-Djacoco.skip=true" flags when we run "verify" in INT. Since in INT we skipped some tests, Jacoco will fail when the coverage is below threshold.